log/sequencer_test.go

```diff
@@ -27,11 +27,11 @@ import (
 	"github.com/google/trillian"
 	"github.com/google/trillian/crypto"
```

Comment:
I thought you'd renamed these to `tcrypto` ?

Reply:
Done.  The preexisting pattern was to rename `"crypto"` to `"gocrypto"`